src/modules/0.0.23/aws_lambda/mappers/lambda_function.ts

```diff
@@ -158,6 +158,55 @@ export class LambdaFunctionMapper extends MapperBase<LambdaFunction> {
     res => res?.NetworkInterfaces,
   );
 
+  async deleteNetworkInterface(client: EC2, subnet: string, name: string) {
+    // iterate until there are no remaining network interfaces
+    let j = 0;
+    do {
+      await new Promise(r => setTimeout(r, 10000)); // Sleep for 10s
+
+      const interfaces = await this.getNetworkInterfaces(client, subnet);
+      let hasDetached = false;
+      if (interfaces) {
+        for (const i of interfaces) {
+          if (i.Description && i.Description.includes(name) && i.NetworkInterfaceId) {
+            if (i.Attachment?.Status !== 'detached') {
+              // need to wait a bit more
+              hasDetached = true;
```

Comment:
hmm, "if no 'detached' then hasDetached"? I'm getting it wrong or the variable name is the opposite?